docs/extending/code-overview.rst

```diff
@@ -3,15 +3,15 @@ Mathics Core Interpreter Overview
 =================================
 
 At its heart, the Mathics interpreter works over `M-expresssion
-<https://en.wikipedia.org/wiki/M-expression>`_: combinations of nested
+<https://en.wikipedia.org/wiki/S-expression>`_: combinations of nested
```

Comment:
I am still not convinced..

Reply:
Ok - so find an example where this is not true.  If you want to demonstrate in the code where head is not assumed to be an Atom, I'd be grateful to learn. (I've been meaning to do this myself but haven't had a chance)

Also, on reflection even _if_ there turn out to be _rare_ situations where this isn't an S-expression, I am leaning towards using that word because it is much more commonly understood. So somewhere _if_ it happens to be a M-expression a footnote somewhere would point this out and just continue with using the more familiar and common idea.